src/native/eventpipe/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ check_symbol_exists(
     HAVE_ACCEPT4)
 
 # Use TCP for EventPipe on mobile platforms
-if (CLR_CMAKE_HOST_IOS OR CLR_CMAKE_HOST_TVOS OR CLR_CMAKE_HOST_ANDROID)
+#
+# TODO: Resolve TCP/IP EventPipe support on CoreCLR Android
```

Comment:
Please file a github issue and record the link here.

Reply:
https://github.com/dotnet/runtime/issues/111491